### ISSUE TITLE
Unwanted tabs re-opening on each browser restart

### ISSUE THREAD

**Issue by gamesguru** [projectdelphai/panorama-tab-groups#239]
Hi, not sure how i achieved this state.

But whenever I open the browser my default tab group gets 5 additional tabs from other groups.  If I do it twice without closing, I will have 10 new tabs in that group.

Weird

**Comment by vaxul** [comment]
I have no idea what could cause this. 

If it's okay for you to loose your browser session and settings, you could refresh your installation to start new.

https://support.mozilla.org/en-US/kb/refresh-firefox-reset-add-ons-and-settings

**Comment by gamesguru** [comment]
Yeah I ended up deleting the offending tabs and a group to be safe, and it went away.  Seems like the group number is on an autoincrement?
